07LessonSeven/homework/index.html

```diff
@@ -1,10 +1,43 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Profession Portal</title>
   <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="CSS/style.css">
```

Comment:
This link here is not needed. Yo have already linked to the style sheet on line 8.